bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-patternfly",
-  "version": "3.2.0",
+  "version": "3.3.0",
   "authors": [
     "Red Hat"
   ],
```

Comment:
Hi,
Need to update:

"dependencies": {
   ...
   ...
    "patternfly": "~3.**3**.0"
  },